gatsby/__mocks__/gatsby.js

```diff
@@ -0,0 +1,27 @@
+const React = require("react")
```

Comment:
muzes prosim vysvetlit co je to a proc to potrebujeme?

Reply:
řídil jsem se oficiálním manuálem: https://www.gatsbyjs.com/docs/unit-testing/#3-useful-mocks-to-complete-your-testing-environment